### ISSUE TITLE
android-apt plugin is not compatible with Android Gradle

### ISSUE THREAD

**Issue by shayan4shayan** [android10/Android-CleanArchitecture#294]
There is a issue when running `./geadlew clean build` 
```
android-apt plugin is incompatible with the Android Gradle plugin.  Please use 'annotationProcessor' configuration instead.
```
I'm using Android studio 3.5 beta and gradle v5.4.1.

**Comment by Arcyno** [comment]
Same problem for me

**Comment by JenniferJohnson89** [comment]
I noticed that there is a similar problem at https://github.com/DigitalCampus/oppia-mobile-android. Perhaps we can refer to https://github.com/DigitalCampus/oppia-mobile-android/issues/643 to find more context about the bug (e.g. the Android Gradle Plugin version). 
Or maybe this can help us find the cause of the bug (android-apt plugin is no longer needed since android gradle plugin 3.0.0) and fix it. After referring to that issue, I tried building using Android Gradle Plugin 2.2.2 and Gradle version 3.2.1, i.e., the original version listed in the project's build.gradle file and gradle-wrapper.properties file. The project builds successfully.
